src/listener/ready.ts

```diff
@@ -28,6 +28,16 @@ export default class ReadyListener extends Listener {
 			this.client.user?.setActivity(`👀 ${memberCount}`, {
 				type: 'WATCHING',
 			});
+
+			const updateStatus = async () => {
+				try {
+					const simpleStats = await this.client.minehutApi.getSimpleStats();
+					if (simpleStats.playerCount == 0 || simpleStats.serverCount == 0) return this.client.user?.setStatus('idle');
```

Comment:
Prettier prettifies this line, have you run Prettier on this file?

Reply:
no

Reply:
Please do so to make the code formatting consistent across the codebase.